.github/workflows/pr.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
       - uses: actions/checkout@v2
       - id: setup-test-java
         name: Set up JDK ${{ matrix.java }} for running tests
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v1.4.3
```

Comment:
Can we revert this? `@v1` means latest version in that major release.

Reply:
Sure...I was just skeptical and experimenting.  I will pull it out.